src/components/SortBar.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+
+export default class SortBar extends Component {
+  render(){
+    return (
+      <div className="sortBox">
+        <div>
+          <button className="sortButton selectedButton">ALL</button>
```

Comment:
Never repeat in JSX similar chanks of code - read about DRY principle
You always should map these similar codes
`{buttonsArray.map( el => {`
  `<button className={IdontKnowWhat}>{el.text}</button>`
  `})}`

Reply:
Resolved.